js/src/forum/components/Search.js

```diff
@@ -88,6 +89,7 @@ export default class Search extends Component {
             className="FormControl"
             type="search"
             placeholder={extractText(app.translator.trans('core.forum.header.search_placeholder'))}
+            aria-label={extractText(app.translator.trans('core.forum.header.search_placeholder'))}
```

Comment:
Should we break this out into a variable? Also, should we consider putting all the aria stuff at the top?

Reply:
Good idea for both